.github/ISSUE_TEMPLATE/z_bug_report.yml

```diff
@@ -0,0 +1,86 @@
+name: Report an issue with Home Assistant Core (Test)
+about: Report an issue with Home Assistant Core.
+title: ""
+issue_body: true
+inputs:
+  - type: description
+    attributes:
+      value: |
+        This issue form is for reporting bugs only!
+
+        If you have a feature or enhancement request, please use the [feature request][fr] section of our [Community Forum][fr].
+        [fr]: https://community.home-assistant.io/c/feature-requests
+  - type: textarea
+    attributes:
+      label: The problem
+      required: true
+      description: >-
+        Describe the issue you are experiencing here to communicate to the
+        maintainers. Tell us what you were trying to do and what happened.
+
+        Provide a clear and concise description of what the problem is.
+  - type: description
+    attributes:
+      value: |
+        ## Environment
+  - type: input
+    attributes:
+      label: What is version of Home Assistant Core has the issue?
+      required: true
+      placeholder: core-
+      description: >
+        Can be found in the Configuration panel -> Info.
+  - type: input
+    attributes:
+      label: What was the last working version of Home Assistant Core?
+      required: false
+      placeholder: core-
+      description: >
+        If known, otherwise leave blank.
+  - type: dropdown
+    attributes:
+      label: What type of installation are you running?
+      required: true
+      description: >
+        If you don't know, you can find it in: Configuration panel -> Info.
+      choices:
```

Comment:
Can we add "other" and then we can make the bot instantly close it?

Reply:
There is no other? :)